lib/sidekiq/client.rb

```diff
@@ -90,15 +90,15 @@ def push(item)
     # Returns an array of the of pushed jobs' jids.  The number of jobs pushed can be less
     # than the number given if the middleware stopped processing for one or more jobs.
     def push_bulk(items)
-      arg = items["args"].first
-      return [] unless arg # no jobs to push
-      raise ArgumentError, "Bulk arguments must be an Array of Arrays: [[1], [2]]" unless arg.is_a?(Array)
+      arg = items["args"]
+      return [] unless arg.present? # no jobs to push
```

Comment:
Nope, this is a Rails method.

Reply:
Apologies, you caught me before I could force push after immediately realizing my mistake 😄

I had tested in my local before pushing, and this did not error out, since `activesupport` was available.

I have now changed this to `.empty?` 🙂